src/coreclr/vm/typedesc.inl

```diff
@@ -30,7 +30,7 @@ inline PTR_MethodTable  TypeDesc::GetMethodTable() {
     if (GetInternalCorElementType() == ELEMENT_TYPE_VALUETYPE)
         return dac_cast<PTR_MethodTable>(asParam->m_Arg.AsMethodTable());
     else
-        return(asParam->GetTemplateMethodTableInternal());
+        return NULL;
```

Comment:
This needs to return `E_T_U` for pointers. It may be best to write it as a switch:

```
inline PTR_MethodTable  TypeDesc::GetMethodTable() {

    LIMITED_METHOD_DAC_CONTRACT;

    switch (GetInternalCorElementType())
    {
    case ELEMENT_TYPE_PTR:
    case ELEMENT_TYPE_FNPTR:
        return CoreLibBinder::GetElementType(ELEMENT_TYPE_U);

    case ELEMENT_TYPE_VALUETYPE:
        return dac_cast<PTR_MethodTable>(dac_cast<PTR_ParamTypeDesc>(this)->m_Arg.AsMethodTable());

    default:
        return NULL;
    }
}
```

Reply:
Ah, right, my bad. Fixed in 3cee32eaba05f2b86b6e041e7b3b78b20d5ded01, thanks! 😄